packages/server/app/Middleware/TemporalAdonisAuth.ts

```diff
@@ -0,0 +1,19 @@
+import { HttpContextContract } from '@ioc:Adonis/Core/HttpContext';
+import Env from '@ioc:Adonis/Core/Env';
+
+export default class TemporalAdonisAuth {
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏Interesting!